go/.gitignore

```diff
@@ -4,3 +4,5 @@
 /server
 /loadtest
 .idea
+/docs/_build
+
```

Comment:
List ``/source/.doctrees`` to the ``.gitignore`` file and delete the files to prevent being tracked on git.